samtranslator/model/connector_profiles/profiles.json

```diff
@@ -352,6 +352,47 @@
             }
           }
         }
+      },
+      "AWS::Location::PlaceIndex": {
+        "Type": "AWS_IAM_ROLE_MANAGED_POLICY",
+        "Properties": {
+          "SourcePolicy": true,
+          "AccessCategories": {
+            "Read": {
+              "Statement": [
+                {
+                  "Effect": "Allow",
+                  "Action": [
+                    "geo:BatchGetDevicePosition",
+                    "geo:CalculateRoute",
+                    "geo:CalculateRouteMatrix",
+                    "geo:DescribeGeofenceCollection",
+                    "geo:DescribeMap",
+                    "geo:DescribePlaceIndex",
+                    "geo:DescribeRouteCalculator",
+                    "geo:DescribeTracker",
+                    "geo:GetDevicePosition",
+                    "geo:GetDevicePositionHistory",
+                    "geo:GetGeofence",
+                    "geo:GetMapGlyphs",
+                    "geo:GetMapSprites",
+                    "geo:GetMapStyleDescriptor",
+                    "geo:GetMapTile",
+                    "geo:GetPlace",
+                    "geo:ListDevicePositions",
+                    "geo:ListGeofences",
+                    "geo:ListTagsForResource",
+                    "geo:ListTrackerConsumers",
+                    "geo:SearchPlaceIndexForPosition",
+                    "geo:SearchPlaceIndexForSuggestions",
+                    "geo:SearchPlaceIndexForText"
```

Comment:
Reading from [Actions defined by Amazon Location](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonlocation.html#amazonlocation-actions-as-permissions), a READ permission to a `AWS::Location::PlaceIndex` resource only needs the following actions.
```
"geo:DescribePlaceIndex",
"geo:ListPlaceIndexes",
"geo:GetPlace",
"geo:SearchPlaceIndexForPosition",
"geo:SearchPlaceIndexForSuggestions",
"geo:SearchPlaceIndexForText"
```